man/man8/puppet-status.8

```diff
@@ -69,7 +69,7 @@ A "true" response or a low\-level connection error\. When used from the Ruby API
 \fBNOTES\fR
 .
 .IP
-This action requires that the server\'s \fBauth\.conf\fR file allow find access to the \fBstatus\fR REST terminus\. Puppet agent does not use this facility, and it is turned off by default\. See \fIhttp://docs\.puppetlabs\.com/guides/rest_auth_conf\.html\fR for more details\.
+This action requires that the server\'s \fBauth\.conf\fR file allow find access to the \fBstatus\fR REST terminus\. Puppet agent does not use this facility, and it is turned off by default\. See \fIhttps://docs\.puppetlabs\.com/guides/rest_auth_conf\.html\fR for more details\.
```

Comment:
This now has a home at https://docs.puppetlabs.com/puppet/latest/reference/config_file_auth.html#access-control-behavior